x/subscription/types/bitset.go

```diff
@@ -0,0 +1,88 @@
+package types
```

Comment:
will need to preserve the license/copyright notices: https://github.com/willf/bitset/blob/master/LICENSE
(both in the headers of the affected files + one entry in the NOTICE file)

Reply:
done